.jenkins/infrastructure/build_azure_managed_images.Jenkinsfile

```diff
@@ -148,7 +148,8 @@ def buildWindowsManagedImage(String os_series, String image_type, String launch_
                                 --resource-group ${VM_RG_NAME} \
                                 --location ${REGION} \
                                 --name ${VM_NAME} \
```

Comment:
why are you moving from ICX to CFL?

Reply:
This part creates the VM that will ultimately be used to create Azure Gallery Images. As part of the image creation process we generalize the VM and then take a snapshot, so the final image doesn't depend on whether it is using DCsv2 or DCsv3. What matters is that it is using a SGX-enabled VM. So it would be better to use DCSv2 due to the slightly lower cost and higher CPU quota available for it.